db/migrate/20230208020404_populate_category_settings.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class PopulateCategorySettings < ActiveRecord::Migration[7.0]
+  def up
+    execute(<<~SQL)
+      INSERT INTO
+        category_settings(
+          category_id,
+          require_topic_approval,
+          require_reply_approval,
+          num_auto_bump_daily,
+          created_at,
+          updated_at
+        )
+      SELECT
+        category_id,
+        MAX(
+          CASE WHEN (name = 'require_topic_approval')
+          THEN value ELSE NULL END
+        )::boolean AS require_topic_approval,
+        MAX(
+          CASE WHEN (name = 'require_reply_approval')
+          THEN value ELSE NULL END
+        )::boolean AS require_reply_approval,
+        MAX(
+          CASE WHEN (name = 'num_auto_bump_daily')
+          THEN value ELSE NULL END
+        )::integer AS num_auto_bump_daily,
+        NOW() AS created_at,
+        NOW() AS updated_at
+      FROM category_custom_fields
+      WHERE name IN (
+        'require_topic_approval',
+        'require_reply_approval',
+        'num_auto_bump_daily'
+      )
+      GROUP BY category_id;
```

Comment:
This `SELECT` pivots up to three rows from `category_custom_fields` into one.

**Example:**

Given these `category_custom_fields` (note no row for `require_reply_approval`):

<img width="490" alt="Screenshot 2023-02-08 at 10 38 02 AM" src="https://user-images.githubusercontent.com/5259935/217414837-e9a84c07-5654-4109-92d6-cc865c25cf60.png">

the query will result in:

<img width="693" alt="Screenshot 2023-02-08 at 10 38 27 AM" src="https://user-images.githubusercontent.com/5259935/217414870-dccd7375-3474-4006-be6c-349e2ce9ae38.png">